src/containers/forgot-password/ForgotPassword.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { Button, Form, Input, Typography } from 'antd';
+import authClient from '../../auth/authClient';
+
+const { Title } = Typography;
+
+const ForgotPassword: React.FC = () => {
+  const onFinish = (values: any) => {
+    authClient.forgotPassword(values);
+  };
```

Comment:
So like, I definitely started the trend of just putting any for onFinish, but like, can we not :) 

Would it be correct to just make `any` -> `ForgotPasswordRequest`? That may type check easily...